Code/Test_Model_On_Kaggle.py

```diff
@@ -1,15 +1,20 @@
 from torch.utils.data import DataLoader
 
 from KaggleDataset import KaggleDataset
-from testing_common_utils import predict_on_test_data, model_uzeros
+from testing_common_utils import predict_on_test_data, model_uzeros, model_uones
 from training_common_utils import image_preprocessing
 
-kaggle_data = KaggleDataset(
-    csv_file="../Data/sample_labels.csv",
-    root_dir="../Data/images",
-    image_transform=image_preprocessing
-)
 
-kaggle_data_loader = DataLoader(kaggle_data)
+def predict(model_name):
+    kaggle_data = KaggleDataset(
+        csv_file="../Data/sample_labels.csv",
+        root_dir="../Data/sample/images",

```

Comment:
Kshitij and I have this directory in different places.  @zbuckley on you to break the tie

Reply:
Mine lines up with the location above.